integrate/romberg_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integrate
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func TestRomberg(t *testing.T) {
+	const n = 1<<16 + 1
```

Comment:
Too many points. I'd use 1000 at most. See also my comment about `h`.

Reply:
With `1<<8 + 1` and after fixing the bugs all test cases pass with `tol = 1e-14` except for the Sqrt one which needs `1e-4`. The polynomial tests would of course pass already with n=3 or 5.